packages/lab/src/__tests__/__e2e__/stepped-tracker/SteppedTracker.cy.tsx

```diff
@@ -10,6 +10,13 @@ import { checkAccessibility } from "../../../../../../cypress/tests/checkAccessi
 
 const composedStories = composeStories(steppedTrackerStories);
 
+const iconShades = {
```

Comment:
Why are we testing CSS here? Wouldn't a visual regression on Chromatic test this better

Reply:
Add the possible combos to QA stories